src/microbe_stage/Membrane.cs

```diff
@@ -269,10 +271,11 @@ public Vector2 FindClosestOrganelles(Vector2 target)
     {
         // The distance we want the membrane to be from the organelles squared.
         float closestSoFar = 4;
-        Vector2 closest = new Vector2(INVALID_FOUND_ORGANELLE, INVALID_FOUND_ORGANELLE);
+        Vector2 closest = invalidVector;
 
-        foreach (var pos in OrganellePositions)
+        for (var i = 0; i < OrganellePositions.Length; i++)
         {
+            var pos = OrganellePositions[i];
```

Comment:
Why did you change this? The previous handling looked better in my eyes.

More generally I feel you changed quite a lot of files (sometimes with good reasons) with no relation to what you wanted to bring, and I feel this makes this PR very dense...

Reply:
This change was one of many tiny performance improvements I've tried to spawning microbes

Reply:
A separate PR is probably warranted as this doesn't fundamentally have anything to do with spawn sectors...